scripts/telegraf.sh

```diff
@@ -15,3 +15,21 @@ start_telegraf() {
 start_host_metrics_telegraf() {
     telegraf --config "$(pwd)/telegraf/telegraf.host.conf"
 }
+
+# Import metrics in file defined by $HOLOCHAIN_INFLUX_FILE into InfluxDB instance via telegraf.
+import_hc_metrics() {
+  if [[ -n ${HOLOCHAIN_INFLUX_FILE:=""} ]]; then
+    if [ -f "$HOLOCHAIN_INFLUX_FILE" ]; then
+      echo Running telegraf for holochain metrics in "$HOLOCHAIN_INFLUX_FILE"
+      # Split huge file otherwise metrics buffer would overflow and metrics get dropped.
+      split -l 50000 "$HOLOCHAIN_INFLUX_FILE" "$WT_METRICS_DIR"/holochain_ -d --additional-suffix=.influx
```

Comment:
In the readme the holochain metrics file is set to a file in WT_METRICS_DIR. Does split remove the original file? Otherwise this would keep the file there and would update some of the metrics from that file too.